### PR TITLE
Improve models

### PR DESCRIPTION
**Pull request by marickmanrho**
Removes:

- models from core folder
- signal from movement and user model

Adds:
- seperate model folder
- create new signal model

Fixes:
- #55

### CODE REVIEW

**Comment by Drvanon** [comment]
First: I really like the new naming convention. It suits the situation a lot better. Second: I really like the smaller PRs it is just so much nicer to look at. Third: I really want the CI on these PRs, I find myself looking through the code, wanting to write a comment and needing to check it out locally first to run the tests, it'd be so much nicer to have that just straight up here. Fourth: I am getting a little sick of all the white space correction in these commits. I think they are redundant. Instead, we should just have one big correction of all files concerning white space and then deploy the new changes on top of that.

**Comment by Drvanon** [comment]
The tests have come back positive, so I think I will accept this branch, but will then ask you to do the white space correction immediately after this.

**Comment by Drvanon** [comment]
Could you move ServerMessage from api.core.ts also to a separate model?

**Comment by marickmanrho** [comment]
Sure thing, just wait with the merger then. I'll update right away.

**Comment by Drvanon** [comment]
Question: we were thinking of splitting Identity and Leader up in different interfaces. Do you want to do that in this PR or in another one?

**Comment by marickmanrho** [comment]
Another, The Identity and Leader are not yet relevant at this stage. Will add at later stage.

File: gridt/src/app/core/models/credentials.model.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export interface Credentials {
+  username: string;
+  password: string;
+}
+
+export interface AccessToken {
+  access_token: string;
+}
```
Comment (Drvanon) [inline gridt/src/app/core/models/credentials.model.ts:8]: I would recommend referring to this file with "server-responses", as it describes both

File: gridt/src/app/core/models/servermessage.model.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export interface ServerMessage {
+  message: string;
+}
```
Comment (Drvanon) [inline gridt/src/app/core/models/servermessage.model.ts:3]: If you will make the "server-response" file, this should be included there.

**Review by Drvanon — CHANGES_REQUESTED** [review]
I think I would like to see it organized in a specific way, but am open to your opinion.

File: gridt/src/app/core/models/credentials.model.ts (hunk 1 above)
Comment (marickmanrho) [reply gridt/src/app/core/models/credentials.model.ts:8]: I did change this, but left Credentials seperate as it is not a server-response.

**Comment by marickmanrho** [comment]
Should be good now.

**Review by Drvanon — APPROVED** [review]
Great. Merging it.

**Pull request merged**